### DISCUSSION TITLE
index.html equivalent

### DISCUSSION THREAD

**Discussion by vispaduchuri** [vercel/next.js#12312] [Help] [answered]
What is the equivalent of index.html in React for next.js? I would like to follow the steps shown in https://github.com/googleads/videojs-ima#es6-imports where they suggest to add certain scripts and link tags along with a Video-js tag?  currently added these tags in the custom _document.js

**Comment by leerob** [comment, marked as answer]
The equivalent of `index.html` in vanilla React (or Create React App) is a new route at `pages/index.js`.
